### PR TITLE
Update resource description field to a WYSIWYG editor in the admin.

### PR DESCRIPTION
**Pull request by Afani97**
What's changed:
- Add `django-ckeditor` package and update requirements
- Update `ResourceAdmin` class with form that uses `CKEditorWidget` for the description field

Preview:
Screenshot 2023-10-25 at 7 15 18 PM

### CODE REVIEW

File: traffic_stops/settings/base.py [hunk 1]
```diff
@@ -357,3 +358,5 @@ def __init__(self, tz_name=None):
 
 # Contact us Email
 CONTACT_US_EMAILS = os.getenv("CONTACT_US_EMAILS", "forwardjustice-team@caktusgroup.com").split(":")
+
+CKEDITOR_BASEPATH = "/static/ckeditor/ckeditor/"
```
Comment (Afani97) [inline traffic_stops/settings/base.py:362]: Need to check if the deployment runs `collectstatic`

Comment (Afani97) [reply traffic_stops/settings/base.py:362]: @copelco If hard coding the path to `static` ok in prod? I see the Dockerfile sets a `UWSGI_STATIC_MAP` to something different. Or is that totally unrelated?

File: nc/admin.py [hunk 2]
```diff
@@ -70,6 +80,7 @@ class ResourceAdmin(admin.ModelAdmin):
     filter_horizontal = ("agencies",)
     inlines = [InlineResourceFile]
     readonly_fields = ("created_date",)
+    form = ResourceForm
```
Comment (copelco) [inline nc/admin.py:83]: [`formfield_overrides`](https://docs.djangoproject.com/en/4.2/ref/contrib/admin/#django.contrib.admin.ModelAdmin.formfield_overrides) might work here, but not required!

File: traffic_stops/settings/base.py (hunk 1 above)
Comment (copelco) [reply traffic_stops/settings/base.py:362]: I think this should work!

**Review by copelco — APPROVED** [review]
Looks good to me!

**Pull request merged**